pyOCD/flash/flash.py

```diff
@@ -75,6 +75,10 @@ class Flash(object):
     """
     This class is responsible to flash a new binary in a target
     """
+    INIT_FUNC_ERASE = 1
+    INIT_FUNC_PROGRAM = 2
+    INIT_FUNC_VERIFY = 3
```

Comment:
Spell out "INIT_FUNCTION" for all three constants instead of shorting to "INIT_FUNC".

Reply:
fixed